packages/marko-web-theme-default/scss/components/_site-footer.scss

```diff
@@ -1,4 +1,5 @@
 .site-footer {
+  z-index: 1; // needed to prevent reveal ad overlap
```

Comment:
Since the reveal ad is a different package altogether, you can leave the `z-index` here, but remove the comment.